Automattic/wp-calypso
# Post-purchase experience: remove pending-migration sticker

## Proposed Changes

Based on this suggestion: p1730446326656409/1730331964.338199-slack-C0Q664T29  this diff stops the `migration-pending` sticker from being set in the `How do you want to migrate?` step.

## Why are these changes being made?


There's a bug that's been affecting users since this code was deployed. Having the sticker on a site where it doesn't belong will trigger usability issues. More details in the thread linked above.

## Testing Instructions



* Apply this PR
* Navigate through the migration flow until you get to the 
<img width="658" alt="image" src="https://github.com/user-attachments/assets/12cb1f83-b6e5-4218-bccc-cbbef7759236">

 * Verify that no request is being triggered to set the `migration-pending` sticker. This is the request that's being triggered now:
<img width="1168" alt="image" src="https://github.com/user-attachments/assets/7b544181-7583-46ee-b326-9a55e3b676fc">

 * **Verify that navigating via both cards works as expected**

## Pre-merge Checklist
